chainweb.cabal

```
@@ -534,6 +534,7 @@ test-suite chainweb-tests
         , random >= 1.2
         , resource-pool >= 0.3
         , retry >= 0.7
+        , rocksdb-haskell-kadena
```

For idiomatic consistency with the rest of this cabal file

```suggestion
        , rocksdb-haskell-kadena >= 1.1.0
```